Makefile

```diff
@@ -119,8 +119,8 @@ cli_test:
 				--language-code de \
 				--field rdfs:label --field IAO:0000115 \
 				-o tests/tmp/example-augmented.babelon.tsv
-	#export OPENAI_API_KEY="" &&\
-	#babelon translate tests/tmp/example-augmented.babelon.tsv -o tests/tmp/example-translated.babelon.tsv
```

Comment:
Why remove the two lines above?

Reply:
for some reason, `make cli_test` didn't like that line. It errors out stating the file path doesn't exist.